charts/tidb-lightning/values.yaml

```diff
@@ -95,6 +95,9 @@ backend: importer # importer | tidb
 # Specify a Service Account for lightning
 # serviceAccount:
 
+# For TiDB-Lightning v3.0.18+ or v4.0.3+,
+# if we want to log to stdout, we should set `file = "-"`.
 config: |
   [lightning]
   level = "info"
+  file = "-"
```

Comment:
It's better to update here https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-lightning/templates/scripts/_start_lightning.sh.tpl#L46.

Reply:
but the user may want to save the checkpoint into another disk for performance, if we hardcode command line args, it will overwrite the one in this config file.

maybe we can remove this line? https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-lightning/templates/scripts/_start_lightning.sh.tpl#L46.

Reply:
OK, in this case, users can config to print log to files.